wdl2cwl/main.py

```diff
@@ -30,6 +30,83 @@ def get_ram_min(ram_min: str) -> int:
     return int(float(ram_min.strip()) * 1024)
 
 
+def get_command(
+    command: str,
+    unbound: List[str],
+    bound: List[str],
+    input_types: List[str],
+    input_names: List[str],
+) -> str:
+    """
+    Get command to be used in the bash script.
+
+    Need to add sep= and default= for expression placeholders
+    """
+    index = 0
+    new_command = ""
+    start_index = 0
+    end_index = 0
+
+    # continue till the end of the string
+    while index < len(command):
+
+        # if you have ~{
+        if command[index] == "~" and command[index + 1] == "{":
+            start_index = index + 2
+
+            # while loop to find index of }
+            while 1:
+                if command[index] == "}":
+                    end_index = index
+                    break
+                else:
+                    index += 1
+
+            # sub string containing everything inside ~{ and }
+            sub_str = command[start_index:end_index]
+
+            # if sub string has a concatenation
+            if "+" in sub_str:
+                split_str = sub_str.split("+")
+
+                for i in split_str:
+                    if i in input_names:
+                        index = input_names.index(i)
+                        data_type = input_types[index]  # get the data type of the input
+
+                        if data_type == "File":
+                            new_command += "$(inputs." + i + ".path)"
+                        else:
+                            new_command += "$(inputs." + i + ")"
+                    else:
+                        new_command += i.replace('"', "")
```

Comment:
This section is also not touched by the test you have. Can you either remove it, or amend your test to use this WDL feature?